src/Endpoint/EnvelopeDocumentAssignmentsEndpoint.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DigitalCz\DigiSign\Endpoint;
+
+use DigitalCz\DigiSign\Resource\BaseResource;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * @extends ResourceEndpoint<BaseResource>
+ */
+class EnvelopeDocumentAssignmentsEndpoint extends ResourceEndpoint
+{
+    public function __construct(EnvelopeDocumentsEndpoint $parent)
+    {
+        parent::__construct($parent, '/assignments');
+    }
+
+    /**
+     * @return array<string, array<string, string>>
+     */
+    public function get(): array
+    {
+        return $this->parseResponse(
+            $this->getRequest('')
+        );
+    }
+
+    /**
+     * @param array<string, array<string, string>> $body
+     */
+    public function set(array $body): ResponseInterface
```

Comment:
pokud endpoint vrací 204, tak bych tady vracel void... v případě non 2xx stavů to vyhazuje vyjímku